docs/index.md

```diff
@@ -77,6 +77,21 @@ Alternatively, Microsoft's python language server (using .NET Core runtime)
 
 [Instructions here](https://github.com/Microsoft/python-language-server/blob/master/Using_in_sublime_text.md)
 
+Use virtualenv adding the following settings :
+
+```json
+"settings": {
+        "python_interpreter": "/Users/mike/.virtualenvs/my-virtual-env/bin/python",
```

Comment:
I can't see where this setting is used by LSP or Sublime Text ( only the Anaconda package)?

Reply:
Looks like this is from [my comment on #531](https://github.com/tomv564/LSP/issues/531#issuecomment-480292747) :)

The `python_interpreter` part was just part of rest of the config I provided for context; only the `LSP` section applies for this plugin.

Reply:
you're guys.
changed